webpack.mix.js

```diff
@@ -14,6 +14,7 @@ const {mix} = require('laravel-mix');
 mix.js('resources/assets/js/app-layout.js', 'public/js')
     .js('resources/assets/js/designer/main.js', 'public/js/designer')
     .js('resources/assets/js/management/users.js', 'public/js/management')
+    .js('resources/assets/js/management/tasks/index.js', 'public/js/management/tasks')
```

Comment:
Management is the folder for the admin pages. Move this to resources/assets/js/processes/tasks.js

Reply:
Changing to folder processes